lib/Doctrine/Validator/Future.php

```diff
@@ -48,11 +48,11 @@ public function validate($value)
         if (count($e) !== 3) {
             return false;
         }
-        
+
         if (is_array($this->args) && isset($this->args['timezone'])) {
             switch (strtolower($this->args['timezone'])) {
                 case 'gmt':
-                    $now = gmdate("U") - date("Z");
+                    $now = getdate((int) gmdate("U") - (int) date("Z"));
```

Comment:
Down below `$now` is accessed as an array (the parts that `getdate` returns), which would break in the case that `gmt` was the `timezone` arg here.

I did confirm that the `$now` object would be the GMT equivalent of `getdate` called without params:

```
php > var_dump(getdate());
array(11) {
  ["seconds"]=>
  int(32)
  ["minutes"]=>
  int(38)
  ["hours"]=>
  int(12)
  ["mday"]=>
  int(14)
  ["wday"]=>
  int(1)
  ["mon"]=>
  int(5)
  ["year"]=>
  int(2018)
  ["yday"]=>
  int(133)
  ["weekday"]=>
  string(6) "Monday"
  ["month"]=>
  string(3) "May"
  [0]=>
  int(1526323112)
}
php > var_dump(getdate(gmdate("U") - date("Z")));
array(11) {
  ["seconds"]=>
  int(53)
  ["minutes"]=>
  int(38)
  ["hours"]=>
  int(18)
  ["mday"]=>
  int(14)
  ["wday"]=>
  int(1)
  ["mon"]=>
  int(5)
  ["year"]=>
  int(2018)
  ["yday"]=>
  int(133)
  ["weekday"]=>
  string(6) "Monday"
  ["month"]=>
  string(3) "May"
  [0]=>
  int(1526344733)
}
```